server/database/schemas/messages-seed.sql

```diff
@@ -0,0 +1,11 @@
+INSERT INTO messages(sender_id, recipient_id, trade_id, body)
+VALUES('cliuo1dcs000208i9hga217k5', 'cliunrnur000008ms1u5qhzl2', 1, 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.')
```

Comment:
Hi @davidguy3237 I don't see created_at data insertion here for this messages table. Did it work for your test?